client/components/Container.jsx

```diff
@@ -1,17 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Button from './Button';
+import axios from 'axios';
 
 const buttonText = 'SEARCH';
 
 const Container = () => {
+  const [inputValue, setInputValue] = useState('');
+  const [error, setError] = useState('');
+
+  const searchHandler = () => {
+    const apiUrl = `http://localhost:3000/marvel/${inputValue}`;
```

Comment:
You should put the base part of the URL into a constant (ideally, in `.env`) so you can easily configure it.